Libraries/Opc.Ua.Gds.Server.Common/ApplicationsNodeManager.cs

```diff
@@ -417,8 +417,7 @@ protected override NodeState AddBehaviourToPredefinedNode(ISystemContext context
                     activeNode.GetTrustList.OnCall = new GetTrustListMethodStateMethodCallHandler(OnGetTrustList);
                     activeNode.GetCertificateStatus.OnCall = new GetCertificateStatusMethodStateMethodCallHandler(OnGetCertificateStatus);
                     activeNode.StartSigningRequest.OnCall = new StartSigningRequestMethodStateMethodCallHandler(OnStartSigningRequest);
-                    // TODO
-                    //activeNode.RevokeCertificate.OnCall = new RevokeCertificateMethodStateMethodCallHandler(OnRevokeCertificate);
+                    activeNode.RevokeCertificate.OnCall = new RevokeCertificateMethodStateMethodCallHandler(OnRevokeCertificate);
```

Comment:
How do I activate the Method, it appears in the structure, but to me it seems the RevokeCertificateMethodStateMethodCallHandler is not activated, as the activeNode.RevokeCertificate.OnCall could not be set?

Reply:
`activeNode.RevokeCertificate = new RevokeCertificateMethodState(passiveNode);

activeNode.Create(context, passiveNode);`